src/components/LeadSpace/LeadSpace.tsx

```diff
@@ -0,0 +1,147 @@
+import React, { useRef, useState } from "react";
+import cx from "classnames";
+import {
+  IconChevronRight,
+  IconPlayerPause,
+  IconPlayerPlay
+} from "@tabler/icons";
+import { Grid, Column } from "../Grid/Grid";
+import Typography from "../Typography/Typography";
+import Button from "../Button/Button";
+
+type CtaItem = {
+  /**
+   * Link to location
+   */
+  href: string;
+
+  /**
+   * Label that is shown
+   */
+  label: string;
+
+  /**
+   * Chevron
+   */
+  showChevron: boolean;
+};
+
+type LeadSpaceProps = {
+  /**
+   * Bg image
+   */
+  backgroundImage?: string;
+
+  /**
+   * Bg image
+   */
+  videoUrl?: string;
+
+  /**
+   * Leadspace title
+   */
+  title?: string;
+
+  /**
+   * CTA section / Buttons
+   */
+  ctaItems?: CtaItem[];
+};
+
+const LeadSpace = ({
+  backgroundImage,
+  videoUrl,
+  title,
+  ctaItems
+}: LeadSpaceProps) => {
+  const video = useRef(null);
+  const [videoPlay, setVideoPlay] = useState(true);
+  return (
+    <section
+      id="leadspace"
+      className={cx("leadspace", { "leadspace--video-player": videoUrl })}
+      style={{
+        backgroundImage: `url(${backgroundImage})`
+      }}
+    >
+      {videoUrl && (
+        <>
+          <video
+            ref={video}
+            autoPlay
+            muted
+            loop
+            disablePictureInPicture
+            playsInline
+            id="leadspace--video"
+            poster={backgroundImage}
+          >
+            <source src={videoUrl} type="video/mp4" />
+          </video>
+          <button
+            className="leadspace--video-controls"
+            type="button"
+            aria-label={videoPlay ? "Pause video" : "Play video"}
+            onClick={() => {
+              if (videoPlay) {
+                video.current.pause();
```

Comment:
Da du das video als ref verwendest und keinen defaultValue angeben kannst, ist es unweigerlich optional. Ergo musst du für den Fall, dass es null ist vorsorgen

 ```suggestion
                video.current?.pause();
```